packages/code-du-travail-frontend/src/common/ThemeBreadcrumbs.js

```diff
@@ -18,8 +18,12 @@ export const ThemeBreadcrumbs = ({ theme, breadcrumbs = [] }) => {
   const themesCrumbs = [ROOT_CRUMB, ...crumbs].map(({ slug, title }) => (
     <Link
       key={slug}
-      href={`/themes${(slug && `/[${slug}]`) || ""}`}
+      href={{
+        pathname: `/themes${(slug && "/[slug]") || ""}`,
+        query: { slug }
```

Comment:
là le fait d'utiliser un objet avec `query` pour décrire href ça ne va pas faire que si on a un slug on se retrouvera avec quelque chose dans le genre: `/themes/bidule?bidule=bidule`  ?
je ne suis pas certain de saisir
et du coup dans le `as` on n'affiche jamais de `?bidule=bidule` ?

Reply:
@lionelB tu en penses quoi ?

Reply:
I would simplify it with 
```js
const HomeLink = <Link key="home"  href="/" title="Retour à l'accueil"  />
      
[HomeLink, ...crumbs].map(({ slug, title }) => ( <Link
      key={slug}
      href="/themes/[slug]"
      as={`/themes/${slug}`}
    >
      <a title={`Voir le theme ${title}`}>{title}</a>
    </Link>
```

Reply:
je me suis basé sur ce qui était dans search/ResultList :)

Reply:
d'ailleurs j'ai écrit un trucs faux :/ 
plutôt
```
[HomeLink].concat(crumbs.map())
```

Reply:
fixed